CONTRIBUTE.md

```diff
@@ -159,6 +159,24 @@ to the database and use logical decoding to read the transaction log. Several ne
 integration test were added.
 ```
 
+### Code Formatting
+
+This project utilizes a set of code style rules that are automatically applied by the build process.  When contributing, it is a good idea to run a build locally to make sure that all code changes you plan to contribute adhere to this style.
+
+In the event that a pull request is submitted with code style violations, continuous integration will fail the pull request build.  
+
+To fix pull requests with code style violations, simply run the project's build locally and allow the automatic formatting happen.  Once the build completes, you will have some local repository files modified to fix the coding style which can be amended on your pull request.  Once the pull request is synchronized with the formatting changes, CI will rerun the build.
```

Comment:
Perhaps add info or a link on how to use the formatter config in Eclipse? Then ideally one would never have dirty files in the workspace after the build, as it wouldn't have to do any formatting.